go/v1beta1/storage/esutil/client.go

```diff
@@ -67,11 +68,13 @@ type MultiSearchRequest struct {
 type GetRequest struct {
 	Index      string
 	DocumentId string
+	Routing    string
 }
 
 type MultiGetRequest struct {
 	Index       string
 	DocumentIds []string
+	Items       []*EsMultiGetItem
```

Comment:
what happens if both `Items` and `DocumentIds` are specified?  is that okay to do, or should we guard against it?

Reply:
Seems fine. Couldn't find anything in the docs about it, but appear to just be merged in the order that the fields appear in the JSON.